recipes/recipes_emscripten/rustworkx/recipe.yaml

```diff
@@ -0,0 +1,41 @@
+context:
+  name: rustworkx
+  version: 0.17.0a3
+
+package:
+  name: ${{name}}
+  version: ${{ version }}
+
+source:
+- url: https://github.com/IvanIsCoding/rustworkx/releases/download/v0.17.0a3/rustworkx-0.17.0a3.tar.gz
```

Comment:
Again, this is one of the few shortfalls of this PR. I hope the forked alpha is OK, we never tried a pre-release on `rustworkx`'s side